src/log/stdio_backend.cr

```diff
@@ -0,0 +1,40 @@
+# A `Log::Backend` that emits to STDOUT.
+class Log::StdioBackend < Log::Backend
```

Comment:
My idea for `StdioBackend` was to allow usage of colors and maybe to split the output in stdout/stderr depending on the severity. This two ideas don't fit in a plain `IOBackend`.

Maybe a `IOBackend` could be created and use it in the StdioBackend implementation.

Also, I would not use `IOBackend` for logging to files. In a `FileBackend` we will likely want options to rotate files.

So since both scenarios include their own options I would rather avoid creating the `IOBackend` until we can iterate with more examples to get it right.

The implementation of `StdioBackend` can evolve and keep the API simple and aimed to be used only in STDOUT/STDERR.

Reply:
> usage of colors

That seems like the responsibility of a formatter, not the backend.

> split the output in stdout/stderr depending on the severity

That should be implemented with separate backend instances for stderr and stdout and severity filters, not some specific internal behaviour. With such a nice framework for compositing logger backends, why not use it?

> In a FileBackend we will likely want options to rotate files.

Certainly not for a basic implementation. More advanced features could be built upon `IOBackend`.

Currently, there's nothing special about `StdioBackend` that wouldn't fit in a generic `IOBackend`.
For the sake of simplicity, I'd prefer to rename it to `IOBackend`. Should we later need additional features we can add specialized `StdioBackend` and `FileBackend` implementations. But for now there's really no reason for that.

Reply:
done